index.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const app = express();
+const path = require('path');
+
+app.use(express.json())
+
+app.get('/', (req, res) => {
+    res.status(200);
+    res.json({message: 'Welcome to the root URL of the server'})
+})
+
+app.get('/hello', (req, res) => {
+    res.set('Content-Type', 'text/html')
+    res.send('<h1>Welcome to the hello URL of the server</h1>')
```

Comment:
The response should be a JSON object like: 
{
   success: 0/1,
   message: 'your message',
   data: {}
}

Reply:
Done!